src/log/__init__.py

```diff
@@ -2,16 +2,18 @@
 import pathlib
 import time
 
-
 def setup_logger():
-    root_dir = pathlib.Path(__file__).parent.parent.resolve()
+    log_dir = pathlib.Path(__file__).parent.parent.parent.joinpath('logs')
```

Comment:
The extra parent will get us on dir above `src`. That is ok when you "run from code" as you probably do now, but when we'll have it packed and the `src` folder will be essentially the root, this can be a problem.

Reply:
What would you like to see? Should we just put log files in the `log` folder, or would make sense to have two folders, `log` and `logs`? I did opt for one directory outside of `src` and did not consider your point. Let me know and I will make a commit to fix it.

Reply:
I'd go on changing the current log file (with the __init__.py file) to `logger`, and have the log files go into `logs`
Meaning we'll have logger and logs folders - less confusing. wdyt?

Reply:
Sounds good to me, I will update the PR with the recommendations.

Reply:
Resolved.